.github/settings.yml

```diff
@@ -160,8 +160,8 @@ branches:
         # Required. The list of status checks to require in order to merge into this branch
         contexts:
           # TODO Temporarily removing e2e tests from required builds as they are flakey and need to be stabilized first
-          # ['lint-and-test', 'action-semantic-pull-request', 'run-e2e-tests']
-          ['lint-and-test', 'action-semantic-pull-request']
+          # ['lint-and-test', 'action-semantic-pull-request', 'Happo', 'run-e2e-tests']
+          ['lint-and-test', 'action-semantic-pull-request', 'Happo']
```

Comment:
Making the Happo report job required. This means devs will have to read the Happo report and mark things as okay before moving on.